client/app/_components/layout/container/stories/Container.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Heading } from '@/_components/ui'
+import { Container } from '../Container'
+import type { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'layout/Container',
+  component: Container,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  tags: ['autodocs'],
+} satisfies Meta<typeof Container>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: null,
+  },
+}
+
+export const ContainerWithHeading: Story = {
+  args: {
+    children: (
+      <div className='h-[400px]'>
```

Comment:
No need to define 400px! It will break if the inside content is more than that.

<img width="418" alt="Screenshot 2024-03-13 at 2 57 58 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/c6756fc9-bb67-4337-b8fc-189113adc125">